src/bridge/cockpitnetworksamples.c

```diff
@@ -92,14 +89,8 @@ cockpit_network_samples (CockpitSamples *samples)
 
       cockpit_samples_sample (samples, "network.interface.rx", iface_name, bytes_rx);
       cockpit_samples_sample (samples, "network.interface.tx", iface_name, bytes_tx);
-
-      total_rx += bytes_rx;
-      total_tx += bytes_tx;
     }
 
-  cockpit_samples_sample (samples, "network.all.rx", NULL, total_rx);
-  cockpit_samples_sample (samples, "network.all.tx", NULL, total_tx);
```

Comment:
We don't document the available metrics, and a google search for these two didn't turn up anything except our own code. Formally it's API, sure, but it's not even a useful metric really (and differs to what you get from PCP). Under these considerations I felt it is okay to remove it, but if you prefer to keep it, we could do that and just not use it ourselves any more.